package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@snyk/composer-lockfile-parser",
+  "description": "Generate a dep tree given a composer.lock file",
+  "main": "dist/index.js",
+  "scripts": {
+    "test": "npm run lint && npm run unit-test",
+    "unit-test": "tap -Rspec ./test/*.test.ts --timeout=300",
+    "lint": "tslint -p tsconfig.json",
+    "build": "tsc",
+    "build-watch": "tsc -w",
+    "prepare": "npm run build"
+  },
+  "types": "./dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/snyk/snyk-composer-parser.git"
+  },
+  "keywords": [
+    "snyk",
+    "php",
+    "composer",
+    "parser"
+  ],
+  "engines": {
+    "node": ">=4"
+  },
+  "homepage": "https://github.com/snyk/snyk-composer-parser#readme",
```

Comment:
composer-lockfile-parser#readme

Reply:
Oops. Leftover from renaming the repo :)